frontend/src/concepts/pipelines/content/configurePipelinesServer/PipelineDropdown.tsx

```diff
@@ -83,20 +83,14 @@ export const PipelineDropdown = ({
           onClick={onToggle}
           isExpanded={isOpen}
         >
-          <KeyIcon />
+          <OptimizeIcon /> <span style={{ marginLeft: '8px' }}>Autofill from connection</span>
```

Comment:
```suggestion
          <Flex spaceItems={{ default: 'spaceItemsSm' }}>
            <FlexItem>
              <OptimizeIcon />
            </FlexItem>
            <FlexItem>
              Autofill from connection
            </FlexItem>
          </Flex>
```

Can we use Patternfly styling here?

Reply:
sure.  i was using cursor to do the margins